pkg/api/pagination_test.go

```diff
@@ -19,6 +19,10 @@ func (p paginationTestObject) EndpointURL(ctx context.Context) (*url.URL, error)
 	return nil, ErrOperationNotSupported
 }
 
+func (paginationTestObject) GetIdentifier(context.Context) (string, error) {
+	return "", ErrObjectWithoutIdentifier
```

Comment:
not that it's actually important here, but return the Message as Identifier instead of an error